packages/angular_devkit/build_angular/builders.json

```diff
@@ -16,6 +16,11 @@
       "schema": "./src/dev-server/schema.json",
       "description": "Serve a browser app."
     },
+    "deploy": {
```

Comment:
Adding a new builder to build-angular should be a separate feature commit.

Reply:
Done